<launch>

    <node pkg="navigation_system" type="NavigationSystem_Node" name="NavigationSystem" output="screen" >
    	<param name="imu_enable" type="bool" value="true" />
    	<param name="mag_enable" type="bool" value="true" />
    	<param name="attitude_enable" type="bool" value="true" />
    	<param name="log_file" type="int" value="5" />
    </node>

</launch>
